<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hollow Scorpion - тестовое задание для Ivideon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/scorpion.png</normaloff>:/scorpion.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="openImagePushButton">
       <property name="text">
        <string>Открыть изображение</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="saltAndPepperGroupBox">
       <property name="title">
        <string>Удалены &quot;Соль и перец&quot; параметры</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QSpinBox" name="saltPepperSpinBox">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>7</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="contoursGroupBox">
       <property name="title">
        <string>Выделение контуров параметры</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSpinBox" name="cannySpinBox">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>7</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="lowThresholdSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="highThresholdSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Удалены &quot;Соль и перец&quot; вывод</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="saltPepperInputLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="saltPepperOutputLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveSaltAntPepperPushButton">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Выделение контуров вывод</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="cannyInputLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cannyOutputLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveContourPushButton">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
